src/components/event_view/event_fields/EventFieldEdit.css

```diff
@@ -0,0 +1,26 @@
+.editSave {
+    margin-right: 2px;
+    font-size: 11px;
+}
+
+.editCancel {
+    font-size: 11px;
+}
+
+.textInput {
+    width: 400px
+}
+
+.dateInput {
+    width: 400px;
+    margin-bottom: 13px;
+}
+
+.currencyInput {
+    width: 400px;
+    margin-bottom: 13px;
+}
```

Comment:
You should avoid redundancy like this with your CSS / each block should be reusable. I would collapse these into one className

Reply:
Think about the "pull-right" class. It's more about what it does than what the element is.

Reply:
This is a super pedantic comment and I don't mind leaving it how it is and I've probably made this mistake too... But if you want to know about CSS best practice and stuff I will just note it... In the same why that Java is usually camel case and c is usually underscores, CSS is usually dashes. Why? Who knows lol.

Reply:
Ahh interesting - didn't know! I think creating a separate issue designated for improving our current use of CSS would be a good idea. That branch could change any syntax errors and also change any React className declarations from `className=""` to Reacts preferred convention` className={''}`.

Reply:
Sounds good. I would definitely be down to do that